site/assets/scss/_component-examples.scss

```diff
@@ -129,12 +131,12 @@
 
 .bd-example-row-flex-cols .row {
   min-height: 10rem;
-  background-color: rgba(255, 0, 0, .1);
+  background-color: rgba(var(--bd-violet-rgb), .1);
```

Comment:
Semi-transparent violets (`rgba(var(--bd-violet-rgb), *`) are difficult to see in dark mode.

![Screenshot 2022-11-25 at 09 38 14](https://user-images.githubusercontent.com/17381666/203937464-0662292d-a311-44bc-bc34-3aade79e927c.png)